ui/static/app/record/record-detail.component.ts

```diff
@@ -190,16 +207,24 @@ export class RecordDetailComponent implements OnInit {
 
     onChangeType() {
       if (this.isCreate) {
-        if (this.record.type === "A" || this.record.type === "MX") {
```

Comment:
if A is select focus set to content field
if MX is select focus set to name field